docs/changelog.rst

```diff
@@ -1,6 +1,11 @@
 Changelog
 =========
 
+5.2 (unreleased)
```

Comment:
I'd use X.Y since it's uncertain what the next version might be.

Reply:
Yes, You are right. just updated!